sw/device/lib/dif/dif_keymgr.h

```diff
@@ -238,6 +247,12 @@ typedef enum dif_keymgr_state {
    * this state, the secret value of key manager is a random value.
    */
   kDifKeymgrStateDisabled,
+  /**
+   * Invalid state.
+   *
+   * Keymgr is in an invalid state and must be reset.
+   */
+  kDifKeymgrStateInvalid,
```

Comment:
Might be worth making this the 0 value.

Reply:
I tried to follow the order in which they are defined in the hw spec.